tonic/src/codec/tests.rs

```diff
@@ -23,7 +23,8 @@ struct Msg {
 async fn decode() {
     let decoder = ProstDecoder::<Msg>::default();
 
-    let data = Vec::from(&[0u8; 1024][..]);
+    let data = Vec::from(&[0u8; 10000][..]);
```

Comment:
You can do this, which I just found out recently :)
```suggestion
    let data = vec![0u8; 10000];
```

Reply:
Oh, sweet.